translations/es/README.md

```diff
@@ -0,0 +1,238 @@
+# Elide
+
+> _API's obstinados para aplicaciones móviles y web._
```

Comment:
What is the difference between obstinados and dogmáticos in Spanish?  Google translate says this is 'stubborn', but I'm no Spanish expert.

Reply:
they are basically the same.
I think the main difference is that dogmáticos refers to both religion or technological **systems** and obstinados is more general, should i switch one of them to keep it consistent?